packages/core/src/renderer.ts

```diff
@@ -16,6 +16,12 @@ import {
   HTTPAction,
 } from "./manifest";
 import { Embed } from "./embeds";
+import Mustache from "mustache";
+
+// Prevent default HTML escaping behaviour
+Mustache.escape = function (value) {
```

Comment:
overriding globals is probably not a good idea given that this code is executed in the same context as other apps